claim-check/code-samples/sample-2/client-consumer/EventHubsConsumer.cs

```diff
@@ -76,36 +35,87 @@ public void Configure()
 
             string storageConnectionString = ConfigurationManager.AppSettings["StorageConnectionString"];
             string eventhubConnectionString = ConfigurationManager.AppSettings["EventHubConnectionString"];
-
             downloadDestination = ConfigurationManager.AppSettings["DownloadDestination"];
-
+            string blobContainerName = ConfigurationManager.AppSettings["BlobContainerName"];
             Console.WriteLine("Connecting to Storage account...");
-
-            storageAccount = CloudStorageAccount.Parse(storageConnectionString);
-
+            BlobContainerClient blobContainerClient = new BlobContainerClient(storageConnectionString, blobContainerName);
             Console.WriteLine("Connecting to EventHub...");
-            
-            client = EventHubClient.CreateFromConnectionString(eventhubConnectionString);
+            processor = new EventProcessorClient(blobContainerClient, EventHubConsumerClient.DefaultConsumerGroupName, eventhubConnectionString);
         }
 
         public async Task ProcessMessages(CancellationToken cancellationToken)
         {
-            Console.WriteLine("The application will now start to listen for incoming message.");
+            Console.WriteLine("The application will now start to listen for incoming messages.");
+
+            Task partitionInitializingHandler(PartitionInitializingEventArgs eventArgs)
+            {
+                if (eventArgs.CancellationToken.IsCancellationRequested)
+                {
+                    return Task.CompletedTask;
+                }
+                try
+                {
+                    var utcNow = DateTime.UtcNow;
+                    eventArgs.DefaultStartingPosition = EventPosition.FromEnqueuedTime(utcNow);
+                    Console.WriteLine($"Initialized partition: { eventArgs.PartitionId }");
+                }
+                catch (Exception ex)
+                {
+                    // For real-world scenarios, you should take action appropriate to your application.  For our example, we'll just log
```

Comment:
nit:  It seems a bit odd to me for there to be a call-out here about example vs. production differences but not in the other similar blocks for the other handlers.  Maybe consider leaving one general comment about the approach used in the handlers before declaring them?